src/components/widgets/accounts-cash-summary/accounts-cash-summary.tmpl.html

```diff
@@ -27,7 +27,7 @@
           </div>
           <div class="row widget-line total" ng-repeat="statement in widget.content.summary" >
             <div class="row widget-line" ng-class="isSelected(statement) ? 'selected' : null" >
-              <div class="col-sm-1" ng-click="toggleCollapsed(statement)"><i class="fa" ng-class="isCollapsed(statement) ? 'fa-plus-square-o' : 'fa-minus-square-o'" /></div>
+              <div class="col-sm-1" ng-click="toggleCollapsed(statement)"><i class="fa" ng-class="isCollapsed(statement) ? 'fa-plus-square-o' : 'fa-minus-square-o'" ng-show="(statement.accounts && statement.accounts.length)"/></div>
```

Comment:
shouldn't the condition be `statement.accounts && statement.accounts.length>0` ?